examples/example_config_files/public_CGD_config.yaml

```diff
@@ -24,7 +24,7 @@ detectors:
   - translate: 
       x: 0
     rotate: 
-      Z: 45
+      Z: 0
```

Comment:
Why? Does this not work with rotated geometries?

Reply:
It does. But you we might need to refine it more. And I wan't to keep the computations in the test at a minimum.